packages/processors/src/processors/strategy/directGrantsSimple/handlers/registered.handler.ts

```diff
@@ -0,0 +1,95 @@
+import { getAddress } from "viem";
+
+import { Changeset, NewApplication } from "@grants-stack-indexer/repository";
+import { ChainId, ProcessorEvent } from "@grants-stack-indexer/shared";
+
+import { IEventHandler, ProcessorDependencies } from "../../../../internal.js";
+import { decodeDGApplicationData } from "../../helpers/index.js";
+
+type Dependencies = Pick<
+    ProcessorDependencies,
+    "roundRepository" | "projectRepository" | "metadataProvider"
+>;
+
+/**
+ * Handles the Registered event for the Donation Voting Merkle Distribution Direct Transfer strategy.
+ *
+ * This handler performs the following core actions when a project registers for a round:
+ * - Validates that both the project and round exist
+ * - Decodes the application data from the event
+ * - Retrieves the application metadata
+ * - Creates a new application record with PENDING status
+ * - Links the application to both the project and round
+ */
+
+export class DGSimpleRegisteredHandler
```

Comment:
```suggestion
export class DirectGrantsSimpleRegisteredHandler
```

may we use complete names for strategies ? or at least use the same for strategy and event handlers :) 

Look here packages/processors/src/processors/strategy/directGrantsSimple/directGrantsSimple.handler.ts

Reply:
[9d6a824](https://github.com/defi-wonderland/grants-stack-indexer-v2/pull/35/commits/9d6a824dd6f7556b247ec858885b8ac1ed1a6194)

i renamed to DGSimple to follow the convention on the other strategies